### DISCUSSION TITLE
MV2 extension disabler exploit

### DISCUSSION THREAD

**Discussion by GameParrot** [3kh0/ext-remover#1207] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Requires chrome://flags to be unblocked.
Most blockers are still MV2. This exploit allows disabling mv2 extensions such as Lightspeed filter agent. To do the exploit, go to chrome://flags, search for extension-manifest-v2-deprecation-disabled, and set it to enabled.

**Comment by schoolexploitkid** [comment]


**Reply by schoolexploitkid** [reply]
nvm, was looking at the wrong flag.

**Comment by HelloIloveGimkit2** [comment]
chrome://flags/#extension-manifest-v2-deprecation-disabled

that's the url

**Reply by HelloIloveGimkit2** [reply]
also don't listen to the dude above it disabled iboss for me
